### ISSUE TITLE
As a developer i want to remove an unused env

### ISSUE THREAD

**Issue by tupe12334** [amplication/amplication#5033]
In the `amplication-build-manager` we have `CREATE_PR_REQUEST_TOPIC` env but we don't use it in the service, we can remove it

**Comment by abrl91** [comment]
@tupe12334 
This env variable is used in prod, and it is best practice to have in your env file the variables that you use in prod
@overbit, what do you think?

**Comment by tupe12334** [comment]
Its not used in the service at all @abrl91

**Comment by abrl91** [comment]
@tupe12334 yes. It is. It was moved to a different repo, but it is in use for this service and that's why I added it

**Comment by tupe12334** [comment]
@abrl91 you are right about  the rest of the values we use in the service, but not in this one
<img width="1512" alt="Screen Shot 2023-01-23 at 10 00 52" src="https://user-images.githubusercontent.com/61761153/213990809-174c7680-cf76-4f14-87ed-ea628df11073.png">

**Comment by abrl91** [comment]
We need to check it carefully as we have this env in prod and it holds the value of the new pull request strategy 
`CREATE_PR_REQUEST_TOPIC: "git.internal.create-pr.request.2"`
